### ISSUE TITLE
Imicus "Upgrade to a New Ship" fit issue.

### ISSUE THREAD

**Issue by UristMcStoutHorn** [ayust/alpha-guide#79]
On this page https://www.thealphasguide.com/upgrading-ships/gallente/imicus/ you have listed "This ship mounts two Warp Core Stabilizer I in its lowslots." however it appears that you can only fit one of these items at a time.

**Comment by ayust** [comment]
Yep, that fit was created before the warp core stabilizer rework that made them an active module. The fit as a whole probably needs to get reworked.

**Comment by UristMcStoutHorn** [comment]
Well thanks for having this whole thing available in the first place.  It's done me wonders thus far.  Then again, I'm only a week old. I know this post doesn't contribute to the ticket really, so feel free to remove it.
